sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -1486,17 +1508,27 @@
             return monoError(logger, new NullPointerException("'subscription' cannot be null."));
         }
 
+        final Context contextWithTrace = context.addData(AZ_TRACING_NAMESPACE_KEY, AZ_TRACING_NAMESPACE_VALUE);
+        final Context contextWithHeaders = contextWithTrace.addData(AZURE_REQUEST_HTTP_HEADERS_KEY, new HttpHeaders());
```

Comment:
I'm not clear why we are flowing this header through? Is it part of a tracing spec?

Reply:
synced up with connie offline, there seems to be no other way of passing around data cleanly, so this is the way forward